exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -121,24 +134,40 @@ class OTLPExporterMixin(
 
     Args:
         endpoint: OpenTelemetry Collector receiver endpoint
+        insecure: Connection type
         credentials: ChannelCredentials object for server authentication
         metadata: Metadata to send when exporting
     """
 
     def __init__(
         self,
-        endpoint: str = "localhost:55680",
+        endpoint: Optional[str] = None,
+        insecure: Optional[bool] = None,
```

Comment:
Missing `protocol`, `certificate_file` in constructor, `compression` and `timeout`?

Reply:
We could add env vars for `protocol` `compression` when those usecases are implemented. Currently exporter uses `grpc` only and no implementation for `compression`.

`certificate_file` is wrapped in ChannelCredentials object which is passed down from `OTLPSpanExporter|MetricExporter`. I'll change `timeout` logic to use env var too.

Reply:
Compression is implemented in a separate PR https://github.com/open-telemetry/opentelemetry-python/pull/1141/files#diff-77db88bbe988c1d7f7d9f92f9e05400259a026e7203f7e9ee12aef76b3b0fdb3R142